env.yml

```diff
@@ -1,12 +1,14 @@
 name: payu
 channels:
   - accessnri
-  - coecms
   - conda-forge
+  - coecms
 dependencies:
+  - python==3.10
   # note that the deployment requires a specific version of 'accessnri::payu' to run
-  - accessnri::payu==1.1.3
-  - coecms::f90nml
+  - accessnri::payu==1.1.4
+  - coecms::yamanifest
+  - f90nml
```

Comment:
Regarding my above comment, if `coecms` is above `conda-forge` in the `channels` list, would it find the `coecms::f90nml` before the `conda-forge::f90nml`? If so, maybe this line should be `- conda-forge::f90nml`, if that's a thing?

Reply:
Yeah so  I think conda won't use f90nml from `coecms` as it has no versions compatible with `python3.10`